### PR TITLE
Add tomli 1.2.2

### PR DESCRIPTION
**Pull request by skupr-anaconda**
Bug Tracker: new open issues https://github.com/hukkin/tomli/issues
Upstream license:  License file:  https://github.com/hukkin/tomli/blob/master/LICENSE
Upstream Changelog: https://github.com/hukkin/tomli/blob/master/CHANGELOG.md
Upstream setup file: https://github.com/hukkin/tomli/blob/1.2.2/pyproject.toml

The package tomli is mentioned inside the packages:
pep517 |

Actions:
1. Fix python
2. Add license_family, dev, doc urls

Result:
- all-fail

### CODE REVIEW

**Comment by skupr-anaconda** [comment]
```
conda.exceptions.ResolvePackageNotFound:
  - flit-core=3.3.0
```

**Comment by skupr-anaconda** [comment]
Added flit-core==3.3.0 on defaults

File: recipe/meta.yaml [hunk 1]
```diff
@@ -17,10 +17,10 @@ build:
...
 requirements:
   host:
-    # Cannot use flit-core >3.3.0 due to cyclic dependency
+    # Cannot use flit-core >3.3.0 due to cyclic dependency 
     - flit-core 3.3.0
     - pip
-    - python >=3.6
+    - python
   run:
     - python >=3.6
```
Comment (katietz) [inline recipe/meta.yaml:26]: I think a skip section for py<36 makes sense here

**Review by katietz — CHANGES_REQUESTED** [review]

File: recipe/meta.yaml (hunk 1 above)
Comment (skupr-anaconda) [reply recipe/meta.yaml:26]: Done

**Pull request merged**